### PR TITLE
Percentiles for describe

### PR DESCRIPTION
**Pull request by cristineguadelupe**

### CODE REVIEW

File: lib/explorer/data_frame.ex [hunk 1]
```diff
@@ -4443,10 +4444,10 @@ defmodule Explorer.DataFrame do
   """
   @doc type: :single
   @spec describe(df :: DataFrame.t()) :: DataFrame.t()
-  def describe(df) do
+  def describe(df, percentiles \\ @default_percentiles) do
```
Comment (josevalim) [inline lib/explorer/data_frame.ex:4447]: Let's make this an option, as we will likely have other percentiles in the future!

Comment (josevalim) [reply lib/explorer/data_frame.ex:4447]: Note we only need to make it an option here. It does not have to be an option in the behaviour.

**Review by philss — APPROVED** [review]
:shipit:

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**